lib/cli/cmd-init.js

```diff
@@ -19,10 +19,14 @@ module.exports = function(folder, opts) {
       console.log('Initializing new DoneJS application at', folderPath);
       console.log('Installing donejs-cli');
 
-      return installCli(mypkg.version, { cwd: folderPath })
-        .then(function() {
-          return runCliInit(folderPath, opts);
-        });
+      if (!fs.existsSync(path.join(folderModules, 'donejs-cli'))) {
```

Comment:
Yes, there is a reason: Essentially, if you let `installCli` run after having done `npm link`, that will for some reason remove some of the deps from the linked `donejs/cli`'s `node_modules`, such as `q`. I'm still not entirely sure why the `npm install` command from `installCli` causes this to happen in the linked package, but turned out not to be something to have to worry about since I figured I could add this if check. Was planning on adding tests for this today, but hadn't really thought about what tests were needed yet, so thanks for the suggestion here.